### PR TITLE
updt model config; add attn config

### PR DESCRIPTION
**Pull request by vchiley**
- updt model config
- adds attn config
- adding `from llmfoundry.models.utils import init_empty_weights` to main as well

### CODE REVIEW

File: llmfoundry/models/mosaic_gpt/configuration_mosaic_gpt.py [hunk 1]
```diff
@@ -51,27 +58,30 @@ def __init__(
             d_model (int): The size of the embedding dimension of the model.
             n_heads (int): The number of attention heads.
             n_layers (int): The number of layers in the model.
-            mlp_ratio (int): The ratio of the up/down scale in the MLP.
+            expantion_ratio (int): The ratio of the up/down scale in the MLP.
```
Comment (abhi-mosaic) [inline llmfoundry/models/mosaic_gpt/configuration_mosaic_gpt.py:61]:
```suggestion
            expansion_ratio (int): The ratio of the up/down scale in the MLP.
```

Comment (abhi-mosaic) [reply llmfoundry/models/mosaic_gpt/configuration_mosaic_gpt.py:61]: Need to find-replace this throughout

Comment (vchiley) [reply llmfoundry/models/mosaic_gpt/configuration_mosaic_gpt.py:61]: 🤦‍♂️ I copy pasta'd that in a few places...
:thankyou:

File: llmfoundry/yamls/mosaic_gpt/testing.yaml [hunk 2]
```diff
@@ -13,10 +13,11 @@ model:
   d_model: 128
   n_heads: 2
   n_layers: 2
-  mlp_ratio: 4
+  expantion_ratio: 4
```
Comment (dakinggg) [inline llmfoundry/yamls/mosaic_gpt/testing.yaml:16]: there a bunch more `mlp_ratio` in the yamls (and also this typo)

Comment (vchiley) [reply llmfoundry/yamls/mosaic_gpt/testing.yaml]: fixed

**Review by abhi-mosaic — APPROVED** [review]
LGTM

**Pull request merged**